topics/MBC_cluster_conversion/mbc_divider_script.py

```diff
@@ -131,6 +133,44 @@ def extract_digits(s):
     return "".join(filter(str.isdigit, s))
 
 
+def safer_drive_scan(drives_container_host_id, old_host_id):
+    retries = 15
+    drives_list = json.loads(run_shell_command('/bin/sh -c "weka cluster drive --host {} -J"'.format(old_host_id)))
+    drives_uuids = [disk["uuid"] for disk in drives_list]
+    print(drives_uuids)
+    scan_disk_command = '/bin/sh -c "weka cluster drive scan {}"'.format(drives_container_host_id)
+    run_shell_command(scan_disk_command)
+    sleep(2)
+    for i in range(retries):
+        drives_list = json.loads(run_shell_command('/bin/sh -c "weka cluster drive --host {} -J"'.format(old_host_id)))
+        drives_uuids = [disk["uuid"] for disk in drives_list]
+        if not drives_uuids:
+            return
+    logger.warning("the following drives did not move to the new container, {} retrying".format(drives_uuids))
+    raise
```

Comment:
`raise Exception("some error msg")` since there is no active exception.
or `quit(1)`